### PR TITLE
tweak input styles

### PR DESCRIPTION
**Pull request by mshwery**
**Overview**
While looking at Autocomplete I noticed some of the font styles were off across all inputs. Fixed!

**Screenshots (if applicable)**
Screen Shot 2020-09-14 at 2 15 08 PM
Screen Shot 2020-09-14 at 2 15 21 PM

**Documentation**

- [ ] Updated Typescript types and/or component PropTypes
- [ ] Added / modified component docs
- [ ] Added / modified Storybook stories

### CODE REVIEW

File: src/themes/classic/components/input.js [hunk 1]
```diff
@@ -14,52 +24,29 @@ const appearances = {
...
     }
   },
   none: {
-    backgroundColor: 'white',
-    _invalid: {},
-    _placeholder: {
-      color: 'scales.neutral.N6A'
-    },
-    _focus: {
-      outline: 'none'
```
Comment (akleiner2) [inline src/themes/classic/components/input.js:57]: Was this just a no-op style?

File: src/themes/classic/components/input.js [hunk 2]
```diff
@@ -14,52 +24,29 @@ const appearances = {
...
     _invalid: {
       boxShadow: theme => `inset 0 0 0 1px ${theme.palette.red.base}`
     },
-    _placeholder: {
-      color: 'scales.neutral.N6A'
-    },
     _focus: {
-      outline: 'none',
       backgroundColor: 'white',
       boxShadow: theme => `0 0 0 2px ${theme.scales.blue.B6A}`
```
Comment (akleiner2) [inline src/themes/classic/components/input.js:42]: Do we need to add the focus z-index here too?

File: src/themes/classic/components/input.js [hunk 3]
```diff
@@ -14,52 +24,29 @@ const appearances = {
...
     _focus: {
-      outline: 'none',
       boxShadow: theme =>
         `inset 0 0 2px ${theme.scales.neutral.N4A}, inset 0 0 0 1px ${theme.scales.blue.B7}, ${theme.shadows.focusRing}`
     },
     _disabled: {
-      cursor: 'not-allowed',
-      boxShadow: theme => `inset 0 0 0 1px ${theme.scales.neutral.N4A}`,
-      backgroundColor: 'scales.neutral.N2'
+      boxShadow: theme => `inset 0 0 0 1px ${theme.scales.neutral.N4A}`
```
Comment (akleiner2) [inline src/themes/classic/components/input.js:32]: Should this then be `theme.colors.neutralAlpha.N4A`?

**Review by akleiner2 — APPROVED** [review]

File: src/themes/classic/components/input.js (hunk 1 above)
Comment (mshwery) [reply src/themes/classic/components/input.js:57]: `outline: none` is now included in the internal styles (maybe it should be moved to theme `baseStyles` tho

File: src/themes/classic/components/input.js (hunk 3 above)
Comment (mshwery) [reply src/themes/classic/components/input.js:32]: nice catch!

**Pull request merged**